tests/install_upgrade_operators/must_gather/conftest.py

```diff
@@ -667,6 +667,7 @@ def collected_must_gather_all_images(
     output = run_must_gather(
         target_base_dir=must_gather_tmpdir_all_images,
         flag_names="all-images",
+        timeout=f"{TIMEOUT_50MIN}s",
```

Comment:
i think you should discuss in the team how to address this collection; waiting up to 50 minutes each time for must-gather sounds too much

Reply:
That timeout would be the maximum: The idea is to leave room to the must-gather to be generated. As soon as it is finished, it will return and that timeout will be not applied.

I run several must-gathers and it's taking between 18-25 minutes. I am going to reduce it to 30minutes.